lib/canary/plugs.ex

```diff
@@ -325,6 +353,10 @@ defmodule Canary.Plugs do
     end
   end
 
+  defp get_persisted(opts) do
```

Comment:
Excellent suggestion, I don't know why I did it like this initially, but I favor this style as well.